flash-messages/env.example

```diff
@@ -0,0 +1 @@
+DATABASE_URL="sqlite:flash-messages-dev.db"
```

Comment:
README has instructions for setting up that adds the variable to .env. Also included it in the comment where session secret is set here: https://github.com/codeunion/examples-sinatra/commit/0b0164ac995a34e3fb07d9c37f5f47cfdf0ec040#diff-d6a8d1c56696b729186dd5ccafab6231R17